### PR TITLE
Demonstrate that artifact invalidation works

### PR DESCRIPTION
**Pull request by jonahwilliams**
## Description

Demonstrate that https://github.com/flutter/flutter/issues/41487 is fixed

### CODE REVIEW

**Comment by jonahwilliams** [comment]
@blasten @zanderso This keeps timing out, due to the added test but does demonstrate that artifact invalidation is working. I can work on re-sharding so this works, or we can otherwise rely on the tool unit tests.

**Comment by blasten** [comment]
@jonahwilliams sg. What about we disable it on Windows and file a bug to enable it back?

**Comment by jonahwilliams** [comment]
Filled https://github.com/flutter/flutter/issues/44455

File: dev/devicelab/bin/tasks/apk_invalidation_test.dart [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+      section('Add assets');
+
+      final File pubspec = File(path.join(projectDir.path, 'pubspec.yaml'));
+      String content = await pubspec.readAsString();
+      content = content.replaceFirst(
+        '\nflutter:\n',
+        '\nflutter:\n  assets:\n    - assets/\n',
+      );
+      print(content);
```
Comment (blasten) [inline dev/devicelab/bin/tasks/apk_invalidation_test.dart]: nit: print

**Review by blasten — APPROVED** [review]
LGTM /w nit

**Review by zanderso — APPROVED** [review]

**Pull request merged**